communication/slack-config/usergroups.yaml

```diff
@@ -3,3 +3,28 @@
 usergroups:
   - name: test-infra-oncall
     external: true
+  - name: youtube-admins
+    long_name: YouTube Admins
+    description: YouTube Admin group. Ping for a YouTube-specific issue
+    channels:
+      - sig-contribex
+    members:
+      - jeefy
+      - sarahnovotny
+      - paris
+      - castrojo
+      - mrbobbytables
+      - alejandrox1
+      - jdumars
+      - jbeda
+      - ihor.dvoretskyi
```

Comment:
Match comment with below -- this should be  `idvoretskyi`. (GitHub ID)

Reply:
Derp Fixed, Part Deux.